effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -768,4 +770,12 @@ object Transformer {
 
   def setStackPointer(stackPointer: Operand)(using C: BlockContext) =
     C.stackPointer = stackPointer;
+
+  def escape(scalaString: String): String =
+    scalaString.foldLeft(StringBuilder()) { (acc, c) =>
+      escapeSeqs.get(c) match {
+        case Some(s) => acc ++= s
+        case None => acc += c
+      }
+    }.toString()
```

Comment:
I just noted that the implementation of escape is (with exception of the ml backend) always the same. @dvdvgt do you think it should be shared, or should we just duplicate it?